dropwizard-demo/src/main/java/com/dep/Services/TwitterImplement.java

```diff
@@ -42,32 +44,37 @@ public Status sendTweets(String tweet) throws TwitterException {
     }
 
 
-    public ArrayList<TweetRespons> fetchLatestTweets() {
-
-        ArrayList<TweetRespons> arrayList = new ArrayList<>();
-        List<Status> statuses = null;
+    public ArrayList<TweetResponse> fetchLatestTweets() {
+        ArrayList<TweetResponse> TweetList = new ArrayList<>();
```

Comment:
Variable name doesn't start with capital letter

Reply:
https://confluence.dev.lithium.com/display/DEV/09+-+Naming+Conventions please read and correct all variable names, class names, file names etc

Reply:
addressed the review